Ginger/GingerCoreNET/RunLib/CLILib/CLIDynamicFile.cs

```diff
@@ -0,0 +1,183 @@
+        {
+            if (FileType == eFileType.JSON)
+            {
+                string json = DynamicExecutionManager.CreateDynamicRunSetJSON(solution, runsetExecutor, cliHelper);
+                return json;
+            }
+            else
+            {
+                string xml = DynamicExecutionManager.CreateDynamicRunSetXML(solution, runsetExecutor, cliHelper);
+                return xml;
+            }
+        }
+
+        public void LoadGeneralConfigurations(string content, CLIHelper cliHelper)
+        {
+            if(FileType == eFileType.JSON)
+            {
+                //Dynamic JSON
+                GingerExecConfig exeConfiguration = DynamicExecutionManager.LoadDynamicExecutionFromJSON(content);
+                if (exeConfiguration.SolutionScmDetails != null)
+                {
+                    cliHelper.SetSourceControlType(exeConfiguration.SolutionScmDetails.SCMType.ToString());
+                    cliHelper.SetSourceControlURL(exeConfiguration.SolutionScmDetails.SolutionRepositoryUrl);
+                    cliHelper.SetSourceControlUser(exeConfiguration.SolutionScmDetails.User);
+                    cliHelper.SetSourceControlPassword(exeConfiguration.SolutionScmDetails.Password);
+                    cliHelper.PasswordEncrypted(exeConfiguration.SolutionScmDetails.PasswordEncrypted.ToString());
+                    if (string.IsNullOrEmpty(exeConfiguration.SolutionScmDetails.ProxyServer) == false)
+                    {
+                        cliHelper.SourceControlProxyServer(exeConfiguration.SolutionScmDetails.ProxyServer);
+                        cliHelper.SourceControlProxyPort(exeConfiguration.SolutionScmDetails.ProxyPort);
+                    }
+                }
+                if (!string.IsNullOrEmpty(exeConfiguration.SolutionLocalPath))
+                {
+                    if (System.IO.Directory.Exists(exeConfiguration.SolutionLocalPath))
+                    {
+                        cliHelper.Solution = exeConfiguration.SolutionLocalPath;
+                    }
+                    else
+                    {
+                        Reporter.ToLog(eLogLevel.INFO, string.Format("Solution local path: '{0}' was not found so creating it", exeConfiguration.SolutionLocalPath));
+                        try
+                        {
+                            System.IO.Directory.CreateDirectory(exeConfiguration.SolutionLocalPath);
+                            cliHelper.Solution = exeConfiguration.SolutionLocalPath;
+                        }
+                        catch(Exception ex)
+                        {
+                            Reporter.ToLog(eLogLevel.ERROR, string.Format("Falied to create the Solution local path: '{0}'", exeConfiguration.SolutionLocalPath), ex);
+                        }
+                    }
+                    if (cliHelper.Solution == null)
+                    {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Either remove or fill this block of code.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=4812633)